pipeline/recon/tool_definitions.py

```diff
@@ -1,9 +1,24 @@
+import json
 from pathlib import Path
 
 from .config import tool_paths, defaults
 
 # tool definitions for recon-pipeline's auto-installer
-tools = {
+tools = None
+with open(Path(__file__).parent / ".." / "data" / "tools.json", "r") as file:
```

Comment:
`Path(__file__).parent / ".."`

can be written as 

`Path(__file__).parent.parent`
or
`Path.cwd().parents[1]`

i personally prefer `Path(__file__).parent.parent` for readability, but that's 100% opinion